tutorials/event_sampler_tutorial.ipynb

```diff
@@ -0,0 +1,582 @@
+    "\n",
+    "The metadata in the event-list are then catched from an [`Observation`](https://docs.gammapy.org/dev/api/gammapy.data.Observations.html) object. \n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Setup"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from pathlib import Path\n",
+    "import numpy as np\n",
+    "import copy\n",
+    "import astropy.units as u\n",
+    "from astropy.coordinates import SkyCoord\n",
+    "from gammapy.data import DataStore, GTI, Observation\n",
+    "from gammapy.datasets import MapDataset, MapDatasetEventSampler\n",
+    "from gammapy.maps import MapAxis, WcsGeom, Map\n",
+    "from gammapy.irf import load_cta_irfs\n",
+    "from gammapy.makers import MapDatasetMaker\n",
+    "from gammapy.modeling import Fit\n",
+    "from gammapy.modeling.models import (\n",
+    "    Model, Models, SkyModel, \n",
+    "    PowerLawSpectralModel, \n",
+    "    PointSpatialModel,\n",
+    "    GaussianSpatialModel,\n",
+    "    SkyDiffuseCube\n",
+    ")\n",
+    "from regions import CircleSkyRegion"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Define the Sky model: point-like source\n",
+    "\n",
+    "First of all, let's define a [`Sky model`](https://docs.gammapy.org/dev/notebooks/models.html) for a point-like source centered 0.5 deg far from the Galactic Center and with a power-law spectral shape. We then save the model into a yaml file."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "spectral_model_pwl = PowerLawSpectralModel(index=2,amplitude=\"1e-12 TeV-1 cm-2 s-1\", reference='1 TeV')\n",
+    "spatial_model_point = PointSpatialModel(lon_0= \"0 deg\", lat_0=\"0.5 deg\", frame='galactic')\n",
+    "sky_model_pntpwl = SkyModel(spectral_model=spectral_model_pwl, spatial_model=spatial_model_point)\n",
+    "\n",
+    "models_pntpwl = Models([sky_model_pntpwl])\n",
+    "\n",
+    "file_model = \"point-pwl.yaml\"\n",
+    "models_pntpwl.write(file_model, overwrite=True)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Create the dataset\n",
+    "In this section we show how to create the `Dataset` object needed for the event sampling (for more info about to create `Dataset` objects, please visit this [link](https://docs.gammapy.org/dev/notebooks/analysis_2.html#Preparing-reduced-datasets-geometry)). \n",
+    "\n",
+    "Hereafter, we select the IRF from the South configuration of the CTA DC1 to simulate one observation. We set the pointing position of the simulated field at the Galactic Center and we fix the exposure time to 8 hr. "
+   ]
```

Comment:
These details might reside on the Proposed Approach

Reply:
Okay, I move it.